.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -1,108 +1,252 @@
 
+All actions should require re-authn with a mfa device.
+
 - [ ] Verify when there are no connectors, empty state renders
-- [ ] Verify that creating OIDC/SAML/GITHUB connectors works
-- [ ] Verify that editing  OIDC/SAML/GITHUB connectors works
+- [ ] Verify that creating, editing, and deleting OIDC/SAML/GITHUB connectors works
+- [ ] Verify that login works for Github/SAML/OIDC
 - [ ] Verify that error is shown when saving an invalid YAML
-- [ ] Verify that correct hint text is shown on the right side
+- [ ] Verify that correct side hint text and doc link is shown
 - [ ] Verify that encrypted SAML assertions work with an identity provider that supports it (Azure).
 - [ ] Verify that created GitHub, saml, oidc card has their icons
+- OSS only:
+  - [ ] github is allowed
+  - [ ] saml + oidc are locked behind CTA's
+
 #### Roles
-- [ ] Verify that roles are shown
+
+All actions should require re-authn with a mfa device.
+
 - [ ] Verify that "Create New Role" dialog works
 - [ ] Verify that deleting and editing works
 - [ ] Verify that error is shown when saving an invalid YAML
-- [ ] Verify that correct hint text is shown on the right side
+- [ ] Verify that correct hint text and doc link are shown
+
+#### Enroll New Integration (aka Plugins)
+
+- [ ] All `self-hosted plugins` and `machine id` cards link out to the correct docs
+- [ ] All `no-code integrations` renders form
+- [ ] All integrations (except for AWS) can only be enrolled once, a checkmark should render and cards are not clickable anymore (unless you delete it)
+- Integrations List
+  - [ ] Verify the created integrations are shown in the integration list
+  - [ ] Integrations can be deleted
+- [ ] AWS External Audit Storage renders only for enterprise cloud
+- In OSS for `no-code integrations``:
+  - [ ] AWS OIDC card is rendered
+  - [ ] AWS External Audit Storage is also rendered but locked behind a CTA
+
+#### Enroll new resources using Discover Wizard
+
+Use Discover Wizard to enroll new resources and access them:
+
+- [ ] SSH Server
+- [ ] Self-Hosted PostgreSQL
+- [ ] AWS RDS PostgreSQL
+- [ ] Kubernetes
+- [ ] AWS EKS cluster
+- [ ] Windows Desktop Active Directory
+- [ ] Non-guided cards link out to correct docs
+
+#### Access Lists
+
+Not available for OSS
+
+Admin refers to users with access_list RBAC defined
+
+- [ ] Renders empty state animation highlighting different features of access lsit
+- [ ] Admins can create, delete, edit, and review access list
+- [ ] Owners (WITHOUT access_list RBAC defined) can still review and edit access lists that they are a owner of
+- [ ] Members can only read access list that they are a member of (edit buttons should be disabled)
+- [ ] Non members, owners, or admins should not be shown any access list
+- [ ] Access list reviews due in two weeks or less
+  - [ ] Red dot is rendered on the notification bell on top bar nav and access lists due are listed
+  - [ ] Red dot is also rendered next to the `Access List` navigation
```

Comment:
The red dot tests feel like something that we could do with react testing library and not have to do every quarter, don't you think?